### DISCUSSION TITLE
checkout

### DISCUSSION THREAD

**Discussion by Marc-cherier** [woocommerce/woocommerce#44130] [Checkout Flow]
checkout pages are not translated into French on the front side (on the back side, yes). It's possible to have the pages in French with the short code, but the price display is at the bottom when it should be on the side (it's a problem of where the Div is closed).

**Comment by opr** [comment]
@Marc-cherier thanks for reporting, just to clarify, are you referring to the Checkout Block? Are you saying the Shortcode checkout is working fine but the block isn't?

Any screenshots to share? Thank you.

**Reply by Marc-cherier** [reply]
if you use the default page without a shortcode, the translation on the French side doesn't work. the translation on the back side works. if you use a shortcode, the form is displayed and works. Only the payment section is at the bottom of the page instead of next to the form. I made a short video on youtube: https://youtu.be/obopTFxk8L4

**Reply by sennevdp** [reply]
I am having the same issue; it's impossible to translate the new Checkout and Cart blocks. When using a translation plugin such as Loco Translate, all strings appear to be correctly translated (to Dutch in my case). However, on the Cart and Checkout pages, most strings are still shown in English.
